### PR TITLE
208 unit test names

### PR DESCRIPTION
**Pull request by fionaRust**
Reference issue if exists

Still to do:
- [ ] Check and fix spot data unit tests
- [ ] Fix unit tests for wind downscaling, hopefully this will be covered by  the other technical debt issues to fix wind downscaling unit tests.
- [ ] Change docstrings to '''First sentence'''' format

Testing:
 - [ ] Ran tests and they passed OK

### CODE REVIEW

File: lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ensemble_copula_coupling_utilities.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# -*- coding: iso-8859-1 -*-
```
Comment (bayliffe) [inline lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ensemble_copula_coupling_utilities.py]: You and your editor.... These probably need to be put back.

Comment (fionaRust) [inline lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ensemble_copula_coupling_utilities.py]: Fixed everywhere hopefully

File: lib/improver/tests/nbhood/vicinity/test_ProbabilityOfOccurrence.py [hunk 2]
```diff
@@ -86,11 +86,11 @@ def setUp(self):
     def test_with_realization(self):
         """Test when a realization coordinate is present."""
         expected = np.array(
-            [[1., 1., 0.83333333, 0.66666667, 0.5],
+            [[1, 1., 0.77777778, 0.55555556, 0.33333333],
```
Comment (fionaRust) [inline lib/improver/tests/nbhood/vicinity/test_ProbabilityOfOccurrence.py]: I've had a quick look and I think @gavinevans is probably right and these changes are due to changes in the behaviour of neighbourhooding when we introduced padding at the edges.

Comment (bayliffe) [inline lib/improver/tests/nbhood/vicinity/test_ProbabilityOfOccurrence.py]: Fine.

**Comment by bayliffe** [comment]
Also renamed wind_downscaling's poorly titled test_hc_rc functions that are not independent tests, but rather callable.

**Comment by bayliffe** [comment]
Been replaced by #229

**Pull request closed without merging**